.github/workflows/user-service-ci.yml

```
@@ -123,9 +123,21 @@ jobs:
             echo "### Failed Tests:" >> $GITHUB_STEP_SUMMARY
             echo '```' >> $GITHUB_STEP_SUMMARY
 
-            # Extract test failures
-            grep -A 5 -B 5 "FAILED" test-output.log | grep -E "(FAILED|expected|but was|at )" >> $GITHUB_STEP_SUMMARY || true
+            # Extract test failures with more context
+            grep -A 10 -B 5 "FAILED" test-output.log | head -100 >> $GITHUB_STEP_SUMMARY || true
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)